rainbow-me/browser-extension
# feat: adds token empty state

Fixes BX-293
Figma link (if any): https://www.figma.com/file/hzfwKOddfRmeVO4KUsG9wM/branch/MBIC9y2TyLFVXCQw4LjeUQ/extensions-v2?node-id=6297%3A209036&t=lYEMzGUNpVM1m5NO-0

## What changed (plus any additional context for devs)
- Adds empty state for Tokens tab
- Tokens empty state is shown when the results from `useBalance` are 0


## Screen recordings / screenshots
<img width="353" alt="image" src="https://user-images.githubusercontent.com/3009331/218600636-32258d68-1e93-42e0-b5c9-f3aee407a1d9.png">



## What to test
- Load a wallet with no asset values
  - You should see the above screenshot
- Load a wallet with asset values
  - After loading, you should see token balances 


## Final checklist
